suite-common/suite-utils/src/device.ts

```diff
@@ -37,6 +37,10 @@ export const getConnectedDeviceStatus = (device: TrezorDevice | undefined) => {
 };
 
 export const getStatus = (device: TrezorDevice) => {
+    if (device.status === 'busy') {
```

Comment:
I think givin the `device-busy` the highest prio is correct, if device is in thi state, you can't do anything about it, regardless it is acquired, non-acquired